packages/examples/src/pages/PixelCanvas/components/CanvasEditor/CanvasEditor.css

```diff
@@ -0,0 +1,9 @@
+#canvasEditor {
```

Comment:
Never use id in css

rename this file to `CanvasEditor.module.css`
Then rename `#canvasEditor` to `.CanvasEditor`
and `#pixels` to `.CanvasEditor-Pixels`

Then in `CanvasEditor.tsx`

```tsx
import styles from './CanvasEditor.module.css`
...

        <div className={styles['CanvasEditor']}>
            <div className={styles['CanvasEditor-Pixels']}ref={panelRef}>
                {rows}
            </div>
        </div>
```